workflow/defaults/settings.yaml

```diff
@@ -29,6 +29,7 @@ default_settings: &default_settings
   nco_mode: NO
   use_nco_ecflow_headers: NO
   print_esmf: .false.
+  esmf_profile: .flase.
```

Comment:
typo

Reply:
This typo is fixed. I'm not sure where the indentation issues are, but if you point them out I will at least fix the ones in this PR.